core/templates/css/oppia.css

```diff
@@ -4837,6 +4842,14 @@ div.oppia-issues-learner-action {
     width: 95%;
   }
 }
+
+@media (max-width: 780px) {
+  /* Prevents misalignment of subtopic card in smaller screen sizes */
+  .subtopics-list .subtopic-card {
+    width: 100% !important;
```

Comment:
We typically try to avoid the use of "!important" in our CSS. Could you explain, conceptually, why it is necessary?

Reply:
> We typically try to avoid the use of "!important" in our CSS. Could you explain, conceptually, why it is necessary?

It was added to override the component css for only the subtopics view, but removed it now by adding another layer of scoping in css. Thanks for pointing it out!